### PR TITLE
Bruceka 2 dev

### PR DESCRIPTION
**Pull request by bruceka**
Console Fix

### CODE REVIEW

File: program.cs [hunk 1]
```diff
@@ -5,3 +5,6 @@
 int k = j / q;
 
 Console.Write(k);
+
+// Comment
+Console.Write"This is the line" + k);
```
Comment (brucekaYh) [inline program.cs]:
```suggestion
Console.Write"This is the line" + k);

This line is missing a (   Please fix!
```

**Review by brucekaYh — CHANGES_REQUESTED** [review]

File: program.cs (hunk 1 above)
Comment (bruceka) [reply program.cs]: Good Catch, Will do....

**Review by brucekaYh — APPROVED** [review]
looks good!

**Comment by bruceka** [comment]
just squashing all of these to one

**Pull request merged**